releasenotes/notes/protobuf-min-f6af9a2d5d96f53c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add support for protobuf>=3.0.0.
```

Comment:
"Other"?

As a feature this would imply a minor version bump?

Reply:
Hmm right. Yeah let's do it as other.